src/main/java/core/basesyntax/dto/FruitRecordDto.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.model.Fruit;
+
+public class FruitRecordDto {
+    private Fruit fruit;
+    private long amount;
+    private OperationType operationType;
+
+    public FruitRecordDto(OperationType operationType, String fruitName, long amount) {
+        this.fruit = new Fruit(fruitName, amount);
+        this.amount = amount;
+        this.operationType = operationType;
+    }
+
+    public Fruit getFruit() {
+        return fruit;
+    }
+
+    public OperationType getOperationType() {
+        return operationType;
+    }
+
+    public enum OperationType {
```

Comment:
if this store will start selling TVs, we'll need another enum?

Reply:
no, TV can have the same OpeartionsType (balance, purchase, return, supply)